core/src/main/scala/tech/beshu/ror/accesscontrol/blocks/definitions/ldap/implementations/UnboundidLdapConnectionPoolProvider.scala

```diff
@@ -186,6 +176,31 @@ object UnboundidLdapConnectionPoolProvider extends Logging {
       .asyncBoundary
   }
 
+  private def testBinding(connectionConfig: LdapConnectionConfig) = {
+    val serverSet = createLdapServerSet(connectionConfig)
+    val bindReq = bindRequest(connectionConfig.bindRequestUser)
+    val maxRetries = if (connectionConfig.ignoreLdapConnectivityProblems) 0 else 5
```

Comment:
I'm wondering if five retries are not too much. In case of a typo and default settings (timeouts), the request will take about 50 seconds.

Reply:
yes, the default connection timeout is poor :( WDYT about 2 attempts? Maybe there is no sense to check more?